tests/topotests/ospf_te_topo1/r1/zebra.conf

```diff
@@ -7,6 +7,7 @@ interface r1-eth0
  link-params
   metric 20
   delay 10000
+  max-bw 10e+10
```

Comment:
Can you be consistent with isis_te_topo1 test by setting max-bw to 1e+10 instead of 10e+10 or change isis_te config to set max-bw to 10+10?

Reply:
done